pytorch_lightning/trainer/data_loading.py

```diff
@@ -100,6 +104,10 @@ def _worker_check(self, dataloader: DataLoader, name: str) -> None:
                 f' in the `DataLoader` init to improve performance.'
             )
 
+    def auto_add_worker_init_fn(self, dataloader: DataLoader) -> None:
+        if dataloader.worker_init_fn is None and int(os.environ.get("PL_SEED_WORKERS", "0")):
```

Comment:
```suggestion
        if dataloader.worker_init_fn is None and int(os.environ.get("PL_SEED_WORKERS", 0)):
```

Reply:
I made it because os.environ also returns str